src/composables/transactionList.ts

```diff
@@ -265,15 +191,16 @@ export function useTransactionList({ store }: IDefaultComposableOptions) {
 
     preparedTransactions = uniqBy(preparedTransactions.reverse(), 'hash').reverse();
     const minMicroTime = Math.min.apply(null, preparedTransactions.map((tx) => tx.microTime));
-    tipWithdrawnTransactions.forEach((f) => {
+    response.tipWithdrawnTransactions?.forEach((f) => {
       if (f.microTime
-        && (
-          minMicroTime < f.microTime
-          || (preparedTransactions.length === 0 && minMicroTime > f.microTime))
+          && (
+            minMicroTime < f.microTime
+            || (preparedTransactions.length === 0 && minMicroTime > f.microTime))
       ) {
```

Comment:
If any of the lines is split I suggest to split all of them:
```suggestion
if (
  f.microTime
  && (
    minMicroTime < f.microTime
    || (preparedTransactions.length === 0 && minMicroTime > f.microTime)
  )
) {
```